### PR TITLE
fix a few things

### PR DESCRIPTION
**Pull request by namusyaka**
I have found a few points that should be fixed.

### CODE REVIEW

File: modules/postgresql/outputs.tf [hunk 1]
```diff
@@ -20,12 +20,12 @@ output "instance_name" {
   description = "The instance name for the master instance"
 }
 
-output "instance_address" {
+output "instance_ip_address" {
```
Comment (namusyaka) [inline modules/postgresql/outputs.tf]: This change is reasonable? @morgante

Comment (morgante) [reply modules/postgresql/outputs.tf]: I think we should leave it as is.

Comment (namusyaka) [reply modules/postgresql/outputs.tf]: OK.
We'll be able to deprecate this and rename in the future (probably next major version bump?)

Comment (namusyaka) [reply modules/postgresql/outputs.tf]: Fixed: https://github.com/terraform-google-modules/terraform-google-sql-db/pull/57/commits/865b976daab04259fa953ee9de04bd2d9e156c3a
Issue opened: #58

**Pull request merged**